vendor/github.com/operator-framework/api/pkg/operators/v1/operator_types.go

```diff
@@ -47,6 +47,29 @@ type Components struct {
 	Refs []RichReference `json:"refs,omitempty"`
 }
 
+func (c Components) Len() int {
```

Comment:
I need to cut a release of the https://github.com/operator-framework/api instead of modifying the vendor files directly.

Reply:
No longer required.